backend/server.py

```
@@ -5965,6 +5965,37 @@ async def autonomy_status() -> dict[str, object]:
+                workspace_root=str(ROOT_DIR),
+            )
+            import asyncio as _aio2
+            result_task = await _aio2.wait_for(
+                coord.execute(task_id), timeout=120.0
+            )
+            dispatch_status["ran"] = True
+            dispatch_status["result_status"] = result_task.status
+            dispatch_status["result_error"] = (result_task.error_message or "")[:100]
+        else:
+            dispatch_status["pending_count"] = 0
+    except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)